idp-service/src/main/java/io/mosip/idp/controllers/OpenIdConnectController.java

```diff
@@ -29,13 +29,13 @@ public class OpenIdConnectController {
      * @return
      * @throws IdPException
      */
-    @GetMapping("/userinfo")
+    @RequestMapping(value = "/oidc/userinfo", method = RequestMethod.GET, produces = "application/jwt")
```

Comment:
I think we can also use get mapping to do this.
 @GetMapping(value = "/userinfo" , produces = {"application/jwt"})

Reply:
Fixed